flink-table/flink-table-calcite-bridge/pom.xml

```diff
@@ -45,23 +45,23 @@ under the License.
 			<version>${calcite.version}</version>
 			<exclusions>
 				<!--
-				"mvn dependency:tree" as of Calcite 1.32.0:
-				[INFO] +- org.apache.calcite:calcite-core:jar:1.32.0:compile
-				[INFO] |  +- org.apache.calcite:calcite-linq4j:jar:1.32.0:compile
+				"mvn dependency:tree" as of Calcite 1.33.0:
+				[INFO] +- org.apache.calcite:calcite-core:jar:1.33.0:compile
+				[INFO] |  +- org.apache.calcite:calcite-linq4j:jar:1.33.0:compile
 				[INFO] |  +- org.locationtech.jts:jts-core:jar:1.19.0:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.14.3:compile
-				[INFO] |  +- org.apache.calcite.avatica:avatica-core:jar:1.22.0:compile
+				[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.15.3:compile
+				[INFO] |  +- org.apache.calcite.avatica:avatica-core:jar:1.23.0:compile
 				[INFO] |  +- org.apiguardian:apiguardian-api:jar:1.1.2:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.14.3:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.14.3:compile
+				[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.15.3:compile
+				[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.15.3:compile
 				[INFO] |  +- com.jayway.jsonpath:json-path:jar:2.7.0:runtime
 				[INFO] |  |  \- net.minidev:json-smart:jar:2.4.7:runtime
 				[INFO] |  |     \- net.minidev:accessors-smart:jar:2.4.7:runtime
 				[INFO] |  |        \- org.ow2.asm:asm:jar:9.1:runtime
 				[INFO] |  +- commons-codec:commons-codec:jar:1.15:runtime
+				[INFO] |  +- org.apache.commons:commons-math3:jar:3.6.1:runtime
```

Comment:
check if this collides / overlaps with flink-core?

Reply:
if we need it, it should be relocated

Reply:
hm... looks like there is no commons-math3 dependency in flink-core
here it is output for dependency:tree from flink-core
```
flink/flink-core$ ../mvnw dependency:tree | grep commons
[INFO] +- org.apache.commons:commons-lang3:jar:3.12.0:compile
[INFO] +- org.apache.commons:commons-text:jar:1.10.0:compile
[INFO] +- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] +- org.apache.commons:commons-compress:jar:1.26.0:compile
[INFO] |  \- commons-io:commons-io:jar:2.15.1:compile
[INFO] |  \- commons-cli:commons-cli:jar:1.5.0:test
[INFO] |  |  \- org.junit.platform:junit-platform-commons:jar:1.10.1:test
```